src/modules/charging-import/lib/queries/return-versions.js

```diff
@@ -92,14 +64,9 @@ on conflict(external_id)
   is_summer=excluded.is_summer,
   is_upload=excluded.is_upload,
   returns_frequency=excluded.returns_frequency,
-  date_updated=excluded.date_updated
-
-`;
+  date_updated=excluded.date_updated`;
```

Comment:
should you not end the SQL statement here with a semicolon before closing the back tick? i.e. date_updated=excluded.date_updated**;**`;